CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On X11, fix `ResumeTimeReached` being fired too early.
+- On Windows, drag and drop is now optional and must be enabled with `WindowBuilderExtWindows::with_drag_and_drop(true)`.
```

Comment:
Drag and drop should be opt-out, not opt-in, for the reasons I brought up in https://github.com/rust-windowing/winit/pull/1260#pullrequestreview-324846105.